index.js

```diff
@@ -102,6 +106,7 @@ class PQueue {
 
 	_next() {
 		this._pendingCount--;
+		this.emit('next');
```

Comment:
I don't think `next` is a good name for this. It's not descriptive enough when you just read the even subscription code. Is it the `next` event of what? Maybe `active` or `dequeued` or something.